docs/product/dashboards/widget-builder/index.mdx

```diff
@@ -132,3 +132,17 @@ The option for setting thresholds is only available for [big number visualizatio
 
 ![Set Thresholds Step](/product/dashboards/widget-builder/widget-builder-thresholds.png)
 ![Widget Coloured Indicator Hover](/product/dashboards/widget-builder/widget-builder-thresholds-hover.png)
+
+## Extracted metrics
+
+<Include name="feature-available-for-user-group-early-adopter.mdx" />
+
+If you are building a widget with transaction data, and would like to use event-like filters (eg. transaction.duration:>100ms) or custom tags that you set on your transactions, we will routinely collect metrics to show you accurate count and rates for you.
```

Comment:
```suggestion
If you are building a widget with transaction data, and would like to use event-like filters (eg. `transaction.duration:>100ms`) or custom tags that you set on your transactions, we will routinely collect [metrics](/product/metrics/) to show you accurate counts and rates.
```

I was thinking we should define what we mean by metrics somewhere, since it's not super clear to me how it's different from event data

Reply:
Unfortunately it's very confusing to everyone because we've been building a lot in the last year or so 😄 . What we collect here and powering the performance pages _are_ conceptually 'metrics' but they are stored in unique rows, not really associated with the `/product/metrics` feature we've built, since they're meant to shadow performance data. You can, in theory, use `metrics` to access the metrics mentioned via transactions / performance but the app isn't really setup that way.

Reply:
Hmm, ok let's remove the link to the metrics product then. But can you define metrics here?  Is the difference between metrics and event data that metrics include all data whereas event data is sampled?

Reply:
> Is the difference between metrics and event data that metrics include all data whereas event data is sampled?

That's exactly right, your visualization is only accurate in count etc. when you're using `metrics` data since it's not dynamically sampled.